### PR TITLE
Server-side implementation

### PR DESCRIPTION
**Pull request by matiasah**
As a solution to #141 STI now relies on love.graphics.isCreated() to determine if the window is created (which is not for servers in most cases).

Feel free to criticize the changes, I'll try to re-adapt this commit as much as possible for convenience.

### CODE REVIEW

File: sti/graphics.lua [hunk 1]
```diff
@@ -0,0 +1,169 @@
+local lg = love.graphics
+local isCreated = lg.isCreated
+local graphics = {isCreated = isCreated}
+
+function graphics.newSpriteBatch(...)
+
```
Comment (phobos2077) [inline sti/graphics.lua]: Why so many blank lines? :)

Comment (matiasah) [inline sti/graphics.lua]: I got used to this 'coding style', it helps me visualize the code faster.

**Comment by karai17** [comment]
please adjust the code style to match sti.

**Comment by karai17** [comment]
thanks!

**Pull request merged**